lambdatimes/src/components/Content/Content.js

```diff
@@ -37,7 +44,15 @@ export default class Content extends Component {
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+    // return this.state.cards;
+    if (this.state.selected === "all") {
+      return this.state.cards;
+    } else {
+      const filtered = this.state.cards.filter(i => {
```

Comment:
Make sure your variable names are descriptive, `i` would be more associated with `index` but this is not an index, it is the element, `card` would make more sense here.